### PR TITLE
Fix: #670, Fix:typos, Fix: reST

### PR DESCRIPTION
**Pull request by svx**
Fixes: #670 

Improves: reST

Changes proposed in this pull request:

- Change ``ZMI`` to ``Management Interface``

- Change ``Zope Management Interface`` to ``Management Interface``

- Fix various reST syntax errors

- Fix typo

This is for 5.1 only, please do not cherry-pick into 4.x !
This is supposed to be released with 5.1, not before !

@tkimnguyen since you opened this issue, I added you as reviewer, thanks !

### CODE REVIEW

File: adapt-and-extend/config/content-settings.rst [hunk 1]
```diff
@@ -80,6 +80,6 @@ All in all, this allows sophisticated setups, where some content items just foll
 
    **Defining your own workflows**
 
-   It is no problem to define your own workflows. By default, however, that involves a trip to the :doc:`ZMI Management Interface <management-interface>` which is not incredibly userfriendly, and defining new workflows is a task best left for your Site Administrator or other specialist.
+   It is no problem to define your own workflows. By default, however, that involves a trip to the :doc:`Management Interface <management-interface>` which is not incredibly userfriendly, and defining new workflows is a task best left for your Site Administrator or other specialist.
```
Comment (tkimnguyen) [inline adapt-and-extend/config/content-settings.rst]: "user friendly" should be two words

File: adapt-and-extend/theming/theme_product.rst [hunk 2]
```diff
@@ -159,7 +161,9 @@ But you will need to name them by prefixing the template name by its complete pa
 
 For instance, to override ``colophon.pt`` from plone.app.layout, knowing this template in a subfolder named ``viewlets``, you need to name it ``plone.app.layout.viewlets.colophon.pt``.
 
-.. note:: ZMI > portal_view_customizations is an handy way to find the template path.
+.. note::
+
+   Management Interface > portal_view_customizations is an handy way to find the template path.
```
Comment (tkimnguyen) [inline adapt-and-extend/theming/theme_product.rst]: ..."is a handy way"

File: appendices/glossary.rst [hunk 3]
```diff
@@ -103,12 +103,13 @@ This is a glossary for some definitions used in this documentation and still und
         customizations.
 
     ZMI
-        The *Zope Management Interface*. Zope has a built in Management
-        Interface that is accessible through the web. Accessing is as simple
-        as appending ``/manage`` to your URL, for example:
+        The *Management Interface*.
+        A Management Interface that is accessible through the web.
+        Accessing is as simple as appending ``/manage`` to your URL, for example:
```
Comment (tkimnguyen) [inline appendices/glossary.rst]: "Accessing it is as simple" (missing "it")

File: develop/addons/components/genericsetup.rst [hunk 4]
```diff
@@ -29,7 +29,7 @@ GenericSetup XML files are usually in a ``profiles/default`` folder inside the a
 All run-time through-the-web (:term:`TTW`) configurable items, like viewlets order through ``/@@manage-viewlets`` page, are made repeatable using GenericSetup profile files.
 
 You do not need to hand-edit GenericSetup profile files.
-You can always change the configuration options through Plone or using the Zope Management Interface, and then you export the resulting profile as an XML file, using the *Export* tab in the ``portal_setup`` ZMI tool.
+You can always change the configuration options through Plone or using the Management Interface, and then you export the resulting profile as an XML file, using the *Export* tab in the ``portal_setup`` Management Interface.
```
Comment (tkimnguyen) [inline develop/addons/components/genericsetup.rst]: "in ``portal_setup``, accessible from the Management Interface"

File: develop/addons/components/genericsetup.rst [hunk 5]
```diff
@@ -837,7 +837,7 @@ Note that you are always allowed to use the ``plone`` domain, but if the xml fil
       ...
     </object>
 
-  Note that when you go to the portal_actions tools in the ZMI, you will see an i18n domain specified for each action.
+  Note that when you go to the portal_actions tools in the Management Interface, you will see an i18n domain specified for each action.
```
Comment (tkimnguyen) [inline develop/addons/components/genericsetup.rst]: "tool" should be singular

File: develop/addons/components/interfaces.rst [hunk 6]
```diff
@@ -7,14 +7,12 @@ Introduction
 ============
 
 Interfaces define what methods an object provides.
-Plone extensively uses interfaces to define APIs between
-different subsystems. They provide a more consistent and declarative
-way to define bridges between two different things, when duck-typing
+Plone extensively uses interfaces to define APIs between different subsystems.
+They provide a more consistent and declarative way to define bridges between two different things, when duck-typing
```
Comment (tkimnguyen) [inline develop/addons/components/interfaces.rst]: what is duck-typing?

File: develop/plone/content/archetypes/fields.rst [hunk 7]
```diff
@@ -512,20 +512,17 @@ Example (mfabrik/rstpage/archetypes/fields.py)::
 Automatically generating description based on body text
 --------------------------------------------------------
 
-Below is a sample through-the-web Python Script which
-you can drop to any Plone through Zope Management Interface.
+Below is a through-the-web (ttw) Python Script which you can drop to any Plone through the Management Interface.
```
Comment (tkimnguyen) [inline develop/plone/content/archetypes/fields.rst]: "TTW" (should be capitalized)

Comment (tkimnguyen) [reply develop/plone/content/archetypes/fields.rst]: "you can drop into"

File: develop/plone/content/archetypes/fields.rst [hunk 8]
```diff
@@ -512,20 +512,17 @@ Example (mfabrik/rstpage/archetypes/fields.py)::
...
 This is not perfect, but this is way better than empty description.
 
-This script will provide one-time operation to automatically
-generate content item descriptions based on their body text
+This script will provide one-time operation to automatically generate content item descriptions based on their body text
 by taking the first three sentences.
 
-The script will provide logging output to standard Plone log
-(var/log and stdout if Plone is run in the debug mode).
+The script will provide logging output to standard Plone log (var/log and stdout if Plone is run in the debug mode).
```
Comment (tkimnguyen) [inline develop/plone/content/archetypes/fields.rst]: "in debug mode"

File: develop/plone/content/dynamic_views.rst [hunk 9]
```diff
@@ -309,10 +309,9 @@ Register the view against a marker interface:
          template="view.pt"
        />
 
-* Assign this marker interface to a content item using the Zope Management Interface
-  (:term:`ZMI`, via the Interfaces tab)
+* Assign this marker interface to a content item using the Management Interface, via the Interfaces tab
```
Comment (tkimnguyen) [inline develop/plone/content/dynamic_views.rst]: "Python" should be capitalized

File: develop/plone/content/dynamic_views.rst [hunk 10]
```diff
@@ -332,8 +331,8 @@ Migration script from default view to another
 ==============================================
 
 Below is a script snippet which allows you to change the default view
-for all folders to another type. You can execute the script through
-the :term:`ZMI` as a Python Script.
+for all folders to another type.
+You can execute the script through the Management Interface as a Python Script.
```
Comment (tkimnguyen) [inline develop/plone/content/dynamic_views.rst]: "Script" should not be capitalized (unless you're referring to a "Script (Python)")

File: develop/plone/content/importexport.rst [hunk 11]
```diff
@@ -37,7 +37,7 @@ To export objects from a site to another, do the following:
 
 * Zope .zexp to *youranothersite*/var/instance/import folder
 
-* Go to ZMI root of your another site
+* Go to the Management Interface root of your another site
```
Comment (tkimnguyen) [inline develop/plone/content/importexport.rst]: "your another" should either be "your other" or just "another" (not sure of the context)

**Comment by svx** [comment]
Welcome in the world of 'old documentation' !

File: develop/plone/content/workflow.rst [hunk 12]
```diff
@@ -23,7 +23,7 @@ For more information, see
 Creating workflows
 ------------------
 
-The recommended method is to use the portal_workflow user interface in the Zope Management Interface
+The recommended method is to use the portal_workflow user interface in the Management Interface
```
Comment (tkimnguyen) [inline develop/plone/content/workflow.rst:26]: "use portal_workflow in the Management Interface"

File: develop/plone/content/workflow.rst [hunk 13]
```diff
@@ -102,7 +102,7 @@ Disabling workflow for a content type
 If a content type doesn't have a workflow it uses its parent container security settings.
 By default, content types Image and File have no workflow.
 
-Workflows can be disabled by setting the workflow setting empty in portal_workflow in ZMI.
+Workflows can be disabled by setting the workflow setting empty in portal_workflow in the Management Interface.
```
Comment (tkimnguyen) [inline develop/plone/content/workflow.rst:105]: "by leaving the workflow setting empty"

File: develop/plone/forms/ploneformgen.rst [hunk 14]
```diff
@@ -9,11 +9,15 @@ PloneFormGen
 Introduction
 ------------
 
-*PloneFormGen* is a Plone add-on Product that provides a generic Plone form generator using fields, widgets and validators from Archetypes. Use it to build simple, one-of-a-kind, web forms that save or mail form input.
+*PloneFormGen* is a Plone add-on Product that provides a generic Plone form generator using fields, widgets and validators from Archetypes.
```
Comment (tkimnguyen) [inline develop/plone/forms/ploneformgen.rst:12]: "is a Plone add-on that"

File: develop/plone/functionality/actions.rst [hunk 15]
```diff
@@ -4,14 +4,13 @@ Actions
...
-        Creating and using portal_actions mechanism
+   Creating and using portal_actions mechanism
 
 
 Introduction
 ------------
 
-Plone has concept of actions which refer the end user functionality associated with site
-or content objects:
+Plone has concept of actions which refer the end user functionality associated with site or content objects:
```
Comment (tkimnguyen) [inline develop/plone/functionality/actions.rst:13]: instead of "refer", maybe "connect"?

File: develop/plone/functionality/collections.rst [hunk 16]
```diff
@@ -62,7 +62,7 @@ features like localization and user-friendly titles.
 
 New criteria can be created through-the-web in Site setup -> Collection section.  Click "All fields" to see unenabled portal_catalog criteria.  Later the edited settings can be exported to GenericSetup XML profile using portal_setup tool (no need to create profile XMl files by hand).
 
-portal_catalog indices can be added through-the-web on ZMI portal_catalog tool tabs.
+portal_catalog indices can be added through-the-web on (ttw) through the Management Interface portal_catalog tool tabs.
```
Comment (tkimnguyen) [inline develop/plone/functionality/collections.rst:65]: remove "on" and capitalize "TTW"

File: develop/plone/misc/portal_transforms.rst [hunk 17]
```diff
@@ -102,8 +102,8 @@ MY.PACKAGE/MYPROFILESDIR/PROFILENAME/MY.PACKAGE-PROFILENAME.txt, so that this
 import step is not run for any profile but just for this one.
 
 
-Calling the Import Step Method in ZMI, portal_setup
----------------------------------------------------
+Calling the Import Step Method in Management Interface, portal_setup
+--------------------------------------------------------------------
 
-Goto your site's portal_setup in ZMI, select your registered profile and import
+Goto your site's portal_setup in Management Interface, select your registered profile and import
```
Comment (tkimnguyen) [inline develop/plone/misc/portal_transforms.rst:108]: "Go to"

File: develop/plone/persistency/transactions.rst [hunk 18]
```diff
@@ -112,7 +112,8 @@ Undoing transactions
 =======================
 
 Everything that has happened on Plone site can be undoed through the *Undo*
-tab in ZMI, in site root. By default you can undo latest 20 transactions.
+tab in the Management Interface, in site root.
```
Comment (tkimnguyen) [inline develop/plone/persistency/transactions.rst:115]: "in the site root"

File: develop/plone/searching_and_indexing/indexing.rst [hunk 19]
```diff
@@ -79,7 +79,7 @@ Viewing indexes and indexed data
 Indexed data
 ^^^^^^^^^^^^
 
-You can do this through portal_catalog tool in ZMI.
+You can do this through portal_catalog tool in the Management Interface.
```
Comment (tkimnguyen) [inline develop/plone/searching_and_indexing/indexing.rst:82]: remove "tool"

File: develop/plone/searching_and_indexing/query.rst [hunk 20]
```diff
@@ -10,18 +10,17 @@ Querying
...
+Plone uses the :doc:`portal_catalog </develop/plone/searching_and_indexing/catalog>`
+tool to perform most content-related queries. Special catalogs, like ``reference_catalog``, exist, for specialized and optimized queries.
 
 
 Accesing the ``portal_catalog`` tool
 ====================================
 
-Plone queries are performed using ``portal_catalog`` persistent tool which
-is available as an persistent object at the site root.
+Plone queries are performed using ``portal_catalog`` persistent tool which is available as an persistent object at the site root.
```
Comment (tkimnguyen) [inline develop/plone/searching_and_indexing/query.rst:23]: curious why we have to say it's persistent, and I think generally appending "tool" to portal_* is unnecessary

File: develop/plone/searching_and_indexing/query.rst [hunk 21]
```diff
@@ -650,7 +651,7 @@ To get all catalog brains of certain content type on the whole site::
 
         campaign_brains = self.context.portal_catalog(portal_type="News Item")
 
-To see available type names, visit in portal_types tool in ZMI.
+To see available type names, visit in portal_types tool in Management Interface.
```
Comment (tkimnguyen) [inline develop/plone/searching_and_indexing/query.rst:654]: "in the Management Interface"

Comment (tkimnguyen) [reply develop/plone/searching_and_indexing/query.rst:654]: remove the first "in" and "tool", so "visit portal_types in the Management Interface"

File: develop/plone/searching_and_indexing/query.rst [hunk 22]
```diff
@@ -1064,7 +1065,7 @@ site objects.
 Here is an example how to crawl through Plone content to search HTML
 snippets. This can be done by rendering every content object and check
 whether certain substrings exists the output HTML This snippet can be
-executed through-the-web in Zope Management Interface.
+executed through-the-web in Management Interface.
```
Comment (tkimnguyen) [inline develop/plone/searching_and_indexing/query.rst:1068]: "the Management Interface"

File: develop/plone/security/local_roles.rst [hunk 23]
```diff
@@ -15,7 +15,7 @@ for a folder and its children.
 
 By default Plone has roles like ``Contributor``, ``Reader``, ``Editor``, etc.
 and you can view these on the :guilabel:`Sharing` tab
-and in :term:`ZMI` :guilabel:`Security` tab.
+and in Management Interface :guilabel:`Security` tab.
```
Comment (tkimnguyen) [inline develop/plone/security/local_roles.rst:18]: "the Management Interface"

File: working-with-content/managing-content/ploneformgen/custom_validators.rst [hunk 24]
```diff
@@ -75,7 +83,9 @@ modules
 Using a Python script
 =====================
 
-You'll be frustrated fast if you try to do anything smart in a single TALES expression. If you need to do something more complicated, add a Python Script to your form folder and call it via TALES. For example, if you added a script with the id includesSpam, you could call it with the expression::
+You'll be frustrated fast if you try to do anything smart in a single TALES expression.
```
Comment (tkimnguyen) [inline working-with-content/managing-content/ploneformgen/custom_validators.rst:86]: "frustrated fast" should be "quickly frustrated"

**Review by tkimnguyen — COMMENTED** [review]
Thanks for replacing "ZMI"! I found some minor tweaks to make.

**Comment by tkimnguyen** [comment]
@jensens @svx I was in the middle of reviewing and stopped to sleep :) I finished the review just now but you merged the PR 4 hours ago... not sure how to incorporate the tweaks I suggested.

**Pull request merged**